src/intelligence_layer/core/model.py

```diff
@@ -232,7 +237,7 @@ class LuminousControlModel(ControlModel):
 ### Response:{{response_prefix}}"""
     )
 
-    AllowedModel = Literal[
+    RecommendedModel = [
```

Comment:
the naming of the variable is kind of off. we could go for constant RECOMMENDED_MODELS, but in any case, it should be plural now